packages/TelegramClientTests-Core.package/TCTCTestCore.class/instance/receiveOrderOfEventSatisfying..st

```diff
@@ -0,0 +1,9 @@
+event log querying
+receiveOrderOfEventSatisfying: aBlock
```

Comment:
When I first read this method name I thought it would somehow return the order of events that satisfy this block. Do you think something like `receivePositionOfEventSatisfying:` of `receiveTimeOfEventSatisfying:` would reflect the intent of this method in a better way?

Reply:
Changed to two methods `loggedEventsSatisfying: aBlock` and `chronologicalPositionOf: anEvent`. Does that work?

Reply:
Not sure, see comment below